README.md

```diff
@@ -58,10 +58,11 @@ Read [USAGE.md](USAGE.md) for usage once Packwerk is installed on your project.
 
 ## Ecosystem
 
-You can use these third party tools to enhance your packwerk experience:
+Various third parties have built tooling on top of packwerk. Here's a selection of some that might prove useful:
 
 - https://github.com/bellroy/graphwerk draws a graph of your package dependencies
 - https://github.com/Gusto/packwerk-vscode integrates packwerk into Visual Studio Code so you can see violations right in your editor
+- https://github.com/Gusto/stimpack is a set of conventions and tooling that makes packwerk packages more like Rails engines.
```

Comment:
Thank you for the inclusion here @exterm . Actually I think we want to de-emphasize rails engine similarity as a selling point.

Making a PR to this end in `stimpack`: https://github.com/Gusto/stimpack/pull/3

Our blurb says:
> `stimpack` establishes and implements a set of conventions for splitting up large monoliths built on top of [`packwerk`](https://github.com/Shopify/packwerk). With `stimpack`, new packages' autoload paths are automatically added to Rails, so your code will immediately become usable and loadable without additional configuration.

Can we use this as the blurb? We could also take out the second sentence if you feel like we want something more concise.

Reply:
I'd love to be a little more specific than just "a set of conventions" and I don't think I'm familiar enough with stimpack to summarize what those conventions actually are 🤔 

Other than that, I am on board 😄

Reply:
Yep, that's the basic idea.

Reply:
`- https://github.com/Gusto/stimpack sets up autoloading and more for a flat list of packages.`

WDYT?